server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndexStats.java

```diff
@@ -23,7 +23,7 @@
 public class IndexStats implements Iterable<IndexShardStats> {
 
     // feature was effectively reverted but we still need to keep this constant around
-    public static final NodeFeature REVERTED_TIER_CREATION_DATE = new NodeFeature("stats.tier_creation_date");
+    public static final NodeFeature REVERTED_TIER_CREATION_DATE = new NodeFeature("stats.tier_creation_date", true);
```

Comment:
This can be removed even though its not in 8.x, as it has no impact on Elasticsearch functionality